### PR TITLE
36 refactor make back-end more dry - Back/End - Pakize Bozkurt

### LINKED ISSUE #36: Refactor: Make backend more DRY
## Story
This is just for the code improvements

## Acceptance Criteria

1. [Make it more DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

## Checklist:

- [ ] I have defined the user story
- [ ] I have defined some acceptance criteria
- [ ] I have included any implementation notes
- [ ] I have included all relevant links

### PR DESCRIPTION
**Pull request by PakizeBozkurt**
## Description
I had some errors. I modified the file with comments.

The code consists of four routes implemented using Express.js and PostgreSQL as the database.
Each of these routes uses the jsonwebtoken library to verify the JWT token in the request header, and the pg library to interact with the PostgreSQL database.

I am going to do DRY.
## Related link
#36 
## Checklist:

- [ Y ] My code follows the style guidelines of this project
- [ Y ] I have carefully reviewed my own code
- [ Y ] I have commented my code
- [ Y ] I have updated any documentation

### CODE REVIEW

File: server/authEndpoint.js [hunk 1]
```diff
@@ -9,17 +9,24 @@ const cors = require("cors");
...
 const pool = new Pool({
-	user: "ekalrqar",
-	host: "34.253.160.140",
-	database: "ekalrqar",
-	password: "E9HOySzUKhJGfNC2sGj_MgkaF32DF_p5",
+	user: "postgres",
+	host: "localhost",
+	database: "study_buddies",
+	password: "CYFStudent123",
 	port: 5432,
```
Comment (Nagehan1) [inline server/authEndpoint.js:19]: hi Pakize, if we can move them  to .env it might be better.
You are keeping going well!

**Review by Nagehan1 — APPROVED** [review]

File: server/authEndpoint.js (hunk 1 above)
Comment (PakizeBozkurt) [reply server/authEndpoint.js:19]: 👍🏻

**Review by Nagehan1 — APPROVED** [review]
Thank you Pakize,

**Comment by PakizeBozkurt** [comment]
It's done. 👍🏻

**Pull request merged**